redirects-www.conf

```diff
@@ -1,5 +1,11 @@
 #Redirect rules for specific pages
 
+rewrite ^/pages/brochures/testing_waters.pdf https://www.fec.gov/updates/testing-the-waters-for-possible-candidacy-2019/ redirect;
+rewrite ^/info/publications/fecglossary2009.pdf https://www.fec.gov/help-candidates-and-committees/guides/ redirect;
+rewrite ^/eeo/documents/annual_no_fear_act.pdf https://www.fec.gov/resources/cms-content/documents/annual_no_fear_act.pdf redirect;
+rewrite ^/info/guidance/debt_retirement.shtml https://www.fec.gov/help-candidates-and-committees/handling-loans-debts-and-advances/ redirect;
+rewrite ^/pages/brochures/TestingtheWaters.shtml https://www.fec.gov/updates/testing-the-waters-for-possible-candidacy-2019/ redirect;
+rewrite ^/pages/brochures/internetcomm.shtml https://www.fec.gov/introduction-campaign-finance/understanding-ways-support-federal-candidates/ redirect;
```

Comment:
May I suggest that these www individual redirects be implemented using Wagtail redirects rather than the proxy? That way, the redirect is instant for the www.fec.gov domain and you can maintain the redirect directly within the Wagtail app